deploy-gae-build-task/task.json

```diff
@@ -56,18 +57,34 @@
       "properties": {
         "EditableOptions": "True"
       },
-      "label": "YAML file name",
       "defaultValue": "app.yaml",
       "required": true,
       "helpMarkDown": "The name of the [YAML configuration file](https://cloud.google.com/appengine/docs/flexible/dotnet/configuration-files) to deploy. [app.yaml](https://cloud.google.com/appengine/docs/flexible/dotnet/configuring-your-app-with-app-yaml) or a custom filename defines the deployment of an app. [cron.yaml](https://cloud.google.com/appengine/docs/flexible/dotnet/scheduling-jobs-with-cron-yaml), [dispatch.yaml](https://cloud.google.com/appengine/docs/flexible/dotnet/how-requests-are-routed#routing_with_a_dispatch_file), [dos.yaml](https://cloud.google.com/appengine/docs/standard/python/config/dosref), [index.yaml](https://cloud.google.com/appengine/docs/flexible/go/configuring-datastore-indexes-with-index-yaml) and [queue.yaml](https://cloud.google.com/appengine/docs/standard/python/config/queueref) configure your app."
     },
     {
+      "name": "deployFromImage",
+      "type": "boolean",
+      "label": "Deploy Existing Image",
+      "defaultValue": false,
+      "helpMarkDown": "Deploy an existing docker image stored in [Google Container Registry](https://cloud.google.com/container-registry)."
+    },
+    {
+      "name": "imageUrl",
+      "type": "string",
+      "label": "Image URL",
+      "defaultValue": "gcr.io/$PROJECTID/image-name:image-tag",
+      "required": true,
+      "visibleRule": "deployFromImage = true",
+      "helpMarkDown": "The URL of the image to deploy. Must be from one of the valid [gcr hostnames](https://cloud.google.com/container-registry/docs/pushing-and-pulling#choosing_a_registry_name). Use $PROJECTID to use the project id defined by the connection."
```

Comment:
Nit, is not an URL, is an image name. I know that the `gcloud` parameter is `--image-url` but that is wrong :). I'm being pedantic, I now.

Reply:
Done.